content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,160 @@
+* Around 80 UI related PRs audited in Jenkins core (introduced in Q3 of 2022)
+
+We have more and more CVE with a name (e.g. https://www.jenkins.io/blog/2022/03/31/spring-rce-CVE-2022-22965/[CVE-2022-22965]) to analyze and answer questions about.
+
+Delivery
+
+* General availability for the Jenkins custom rules in CodeQL (https://groups.google.com/g/jenkinsci-dev/c/OMe_zN8-Tkc/m/5Tf0OnNWAgAJ[message])
+* Improved tooling for the SECURITY tickets handling
+
+== Infrastructure Update
+
+Contributed by: https://github.com/dduportal[Damien Duportal]
+
+== Platform Update
+
+Contributed by: https://github.com/gounthar[Bruno Verachten]
+
+* Platform modernization
+** Require Java 11. https://www.jenkins.io/blog/2022/12/14/require-java-11/[Build toolchain changes] arrived in parent pom 4.52
+** Fully supports Java 17
+** Now more Java 11 installations than Java 8 installations of Jenkins core image:/images/post-images/2023/01/10/jenkins-newsletter/image4.png[image,width=403,height=275]
+** Migrate Linux installation packages from System V init to systemd
+** Staying on top of new backend and frontend dependency updates
+** Container images
+*** New platforms support (arm/v7, aarch64, etc.) and removal of ppc64le
+*** Very last, definitive version of the containers for JDK8
+*** Container image deprecation for the blueocean container
+*** Removed the deprecated install-plugins.sh script from Docker images
+*** Exit (and restart) lifecycle change in the Docker images
+** ANTLR 2 to ANTLR 4 transition complete, Jenkins core compiles
+*** Thanks to Alex Earl and Basil Crow!
+*** Included in Jenkins 2.376
+* Platform documentation
+** Short https://www.jenkins.io/doc/administration/requirements/servlet-containers/#sidebar-content[guide] about web containers
+* Platform WIP
+** Experiments with RISC-V agents with JDK17/19/20
+** Experiments with Windows 2022
+
+== User Experience Update
+
+Contributed by: https://github.com/MarkEWaite[Mark Waite]
+
+Jenkins LTS and weekly releases in 2022 have included significant user experience improvements thanks to the work of Jan Faracik, Tim Jacomb, Alex Brandes, Daniel Beck, and many others. Table layouts, menu entries, icons, themes, breadcrumbs, and more have been updated to give Jenkins a new, fresh look in 2022.
+
+== Documentation Update
+
+Contributed by: https://github.com/kmartens27[Kevin Martens]
+
+This year, we saw documentation contributions from new and seasoned Jenkins users.
+These contributions included blog posts, documentation additions and updates, documentation migration, and other improvements.
+All of this combined has helped expand and empower the Jenkins community.
+
+The Jenkins project saw 48 blog posts, submitted by 23 different authors, and overall 814 PRs were merged.
+The contributions are a result of the community and collaboration, with various projects throughout the year such as She Code Africa, Google Summer of Code, and Hacktoberfest.
+
```

Comment:
![Jenkins-site-improvements](https://user-images.githubusercontent.com/99040580/211110471-48eaa0c7-5288-4872-962a-3b81a36cfeca.png)
same as previous suggestions of adding in the header image

Reply:
Done, thanks.